### ISSUE TITLE
Modify api-next's key route for querying fingerprints

### ISSUE THREAD

**Issue by ryyppy** [uselagoon/lagoon#104]
In the `development` branch, `auth-ssh` apparently uses `api-next` to query an existing ssh-key fingerprint, but I didn't adapt the `/keys` route to query the data from the database yet (I thought we discussed that the original `api` will stay for a while before being replaced).

Anyways, steps which need to be done in this file -> https://github.com/amazeeio/lagoon/blob/develop/services/api-next/src/routes/keys.js#L14-L52 👍 

- [x] Add dao functions to replace the original store logic (get all customer ssh keys etc)
- [x] Use the context to retrieve the `dao` functions 
- [x] Use the `toFingerprint` for the ssh keys coming from the database query for the rest of the logic

Long story short: This impl still uses the redux store of the past api service, we need to consume data from the database

Manual test steps:

```
IMAGE_REPO=lagoon docker-compose up auth-ssh api-next auth-server

# make sure you have a certain ssh configuration to use our lagoon test private key etc
# auth-ssh is an entry in my `.ssh/config` file, pointing to 127.0.0.1:2020 and using a
# different private ssh key file
ssh auth-ssh login 
```

**Comment by Schnitzel** [comment]
@ryyppy @karlhorky 
yes correct, originally we thought we might keep the old `api` a bit around, but I think it makes more sense to migrate everything :)
I actually removed the `ssh-auth` test from the Makefile so that we have green tests: https://github.com/amazeeio/lagoon/blob/develop/Makefile#L313 (the ssh-auth is not used in production anywhere yet, so I felt comfortable to disable the test for now)

As soon as this is done, we can add it again.

thanks!

**Comment by ryyppy** [comment]
@Schnitzel I will let you know as soon as I migrated the auth-ssh to the api-next.

Since `api` is already removed from the `develop` codebase, maybe we should consider renaming `api-next` back to `api`? Or will this be too confusing?
